resources/definitions/fdmprinter.def.json

```diff
@@ -3873,6 +3873,19 @@
                         }
                     }
                 },
+                "support_infill_angle":
+                {
+                    "label": "Support Infill Line Direction",
+                    "description": "Orientation of the infill pattern for supports. The support infill pattern is rotated in the horizontal plane.",
+                    "unit": "°",
+                    "type": "float",
+                    "minimum_value": "-90",
+                    "maximum_value": "90",
+                    "default_value": 0,
+                    "enabled": "support_enable",
```

Comment:
Could you please disable the setting if _Support Pattern_ is Concentric or Cross support.

Reply:
Good catch. I've also disabled the setting if the Support Density is set to 0 (in analogy to Infill Line Directions.